src/toolkit/position.h

```diff
@@ -3,11 +3,13 @@
 #define CYCLUS_DECIMAL_SECOND_MULTIPLIER 3600
 
 #include <string>
-
 #include "cyclus.h"
 
+
+
```

Comment:
Three seems like a lot of blank lines at once. (Though, I don't remember the style guide rule on this). 
```suggestion
```